test/functional/app/api/test_search_api.py

```diff
@@ -0,0 +1,74 @@
+"""
+This file tests search_api.py.
+"""
+import json, pytest
+from time import sleep
+from server.utils.auth import get_complete_auth_token
+from server.utils.user import create_transfer_account_user
+from server import db
+
+def test_transfer_usage_api_get(test_client, complete_admin_auth_token, create_organisation):
```

Comment:
this test looks super clean and basically perfect, BUT name of test needs to be renamed `test_transfer_usage_api_get` 👀 and doc string  fixed

Reply:
Don't know how I missed that!